### ISSUE TITLE
About learning rate

### ISSUE THREAD

**Issue by HaixiongLi** [milesial/Pytorch-UNet#117]
Why the learning rate here is set to 0.1, is this learning rate high? Why not use the learning rate decay function

**Comment by milesial** [comment]
0.1 worked quite well for me with Adam, which is adaptative. You can experiment with a decay schedule if you wish, I have not tried that.
